src/main/java/com/axibase/tsd/client/IgnoringTrustManager.java

```diff
@@ -31,4 +33,24 @@ public void checkClientTrusted(X509Certificate[] chain,
                                    String authType) throws CertificateException {
         //No implementation here
     }
+
+    @Override
+    protected Object clone() throws CloneNotSupportedException {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this "clone" implementation; use a copy constructor or copy factory instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS2975)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS1185)